packages/workspace/src/utilities/executor-options-utils.ts

```diff
@@ -14,6 +14,7 @@ type CallBack<T> = (
 
 /**
  * Calls a function for each different options that an executor is configured with
+ * @deprecated This function will be removed from @nrwl/workspace in version 17. Prefer importing from @nrwl/devkit.
```

Comment:
Looks like no one is using this, we can just remove it I think: https://grep.app/search?q=forEachExecutorOptions

Reply:
I need to start using this grep app

Reply:
Its sweet for this kind of thing, just have to take a grain of salt that its only open source projects